### ISSUE TITLE
onyo set throwing error while setting ' ' to key again

### ISSUE THREAD

**Issue by j-leo** [psyinfra/onyo#454]
```
(onyo) jleo@bnbnb70 in ~/onyo-test on git:master
❱ cat dp/Jan\ Leonhardt/laptop_lenovo_ThinkpadP14s.fauxC4D6tW 
type: laptop
make: lenovo
model: ThinkpadP14s
serial: fauxC4D6tW
RAM: 17
Size:
disk: 512
keyboard: de
CPU:
comment: ' '
(onyo) jleo@bnbnb70 in ~/onyo-test on git:master
❱ onyo set -p dp/Jan\ Leonhardt/laptop_lenovo_ThinkpadP14s.fauxC4D6tW -k comment=' '
The following assets will be changed:
Update assets? (y/n) y
ERROR: Command '['git', 'commit', '-m', 'edit: dp/Jan Leonhardt/laptop_lenovo_ThinkpadP14s.fauxC4D6tW\n\n--- Inventory Operations ---\nModified assets:\n- dp/Jan Leonhardt/laptop_lenovo_ThinkpadP14s.fauxC4D6tW\n']' returned non-zero exit status 1.
```

**Comment by j-leo** [comment]
it is more a confusing output (if you are not a git-pro) which always occurs when actually no change shall be committed

**Comment by bpoldrack** [comment]
Fixed in #465
